tools/view.js

```diff
@@ -440,9 +468,6 @@ View.prototype.displaySuspendEvent = function (data, getSource) {
     var annotationArray = sourceToArray(source.sourceText);
     annotateArray(annotationArray, source.id, data.sections);
     sourceFile.html(arrayToString(annotationArray));
-
-    // enable clicking on EventualSendNodes
-    enableEventualSendClicks(sourceFile);
```

Comment:
I thought that the enableEventualSendClicks in showSource function is the one that triggers the popover on first load, so I think is not need it there anymore in displaySuspendEvent function I will test it